test/spec/tokenManager.js

```diff
@@ -0,0 +1,189 @@
+define(function(require) {
+  var OktaAuth = require('OktaAuth');
+  var tokens = require('../util/tokens');
+  var util = require('../util/util');
+
+  function setupSync(options) {
+    options = options || {};
+    return new OktaAuth({
+      url: 'https://auth-js-test.okta.com',
+      tokenManager: {
+        storage: options.type
+      }
+    });
+  }
+
+  beforeEach(function() {
```

Comment:
beforeEach guarantees I start with a clean slate, while afterEach guarantees I finish with a clean slate. With afterEach, I can't be sure that localStorage or sessionStorage is clean after running other spec files.